conntable.go

```diff
@@ -71,20 +74,42 @@ func (t *connTable) Register(ck ckey, matchedRule *Rule, srcIP, srcPort string,
 			TargetPort: targetPort,
 			//TargetIP:   targetIP,
 		}
+
 	}
 }
 
-func (t *connTable) FlushOlderThan(s time.Duration) {
+func (t *connTable) FlushOlderOnes() {
 	t.mtx.Lock()
 	defer t.mtx.Unlock()
 
-	threshold := time.Now().Add(-1 * s)
+	for minutes := 5; minutes >= 0; minutes -= 1 {
```

Comment:
Second comment: I think the cleanup logic needs to be re-worked at a higher level in a future commit/release.

The current structure is hard for me to reason about:

* Invoke cleanup every **X** seconds
* Exit cleanup if less than **Y** connections
* Ignore connection if it is less than **Z** seconds old

There is no guarantee that the cleanup routine will actually succeed.

Also, if we are strict about the cleanup, then we need to be equally strict about handling packets for connections that have been aged out.